sandpack-react/src/contexts/sandpackContext.tsx

```diff
@@ -463,8 +465,8 @@ export class SandpackProviderClass extends React.PureComponent<
     }
   };
 
-  setActiveFile = (activeFile: string): void => {
-    this.setState({ activeFile });
+  setActiveFile = (activeFile: string, initialCursor?: Cursor): void => {
+    this.setState({ activeFile, initialCursor });
```

Comment:
Shouldn't we clean the `initialCursor` after opening the file? I was wondering if it might affect the next file and use a stale value.